pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -782,7 +782,7 @@ public SuccessResponse reloadAllSegments(
         LOGGER.error("Failed to add reload all segments job meta into zookeeper for table: {}", tableNameWithType, e);
       }
     }
-    return new SuccessResponse("Segment reload details: " + JsonUtils.objectToString(perTableMsgData));
+    return new SuccessResponse(JsonUtils.objectToPrettyString(perTableMsgData));
```

Comment:
It doesn't need to be a json, maybe simply
```suggestion
    return new SuccessResponse("Segment reload details: " + perTableMsgData);
```

Reply:
it doesn't need to be, but what value is `Segment reload details:` to the result of an endpoint that is clearly already the reload details. The impetus for this change was to make it easier to parse the result in java/scala so you don't have to go from request -> strip the prefix -> decode the json string -> decode the json. You should be able to go from request -> json in 1 hop.